packages/fx-core/src/common/local/taskDefinition.ts

```diff
@@ -103,6 +103,45 @@ export class TaskDefinition {
     };
   }
 
+  static funcHostedBotWatch(workspaceFolder: string): ITaskDefinition {
+    return {
+      name: "bot watch",
+      // command is never used for the new local preview in CLI.
+      command: "npm run watch:teamsfx",
+      cwd: path.join(workspaceFolder, FolderName.Bot),
+      execOptions: {
+        needShell: true,
+        needCmd: isWindows(),
```

Comment:
curious why `needCmd` is required?

Reply:
copied from backend start
![image](https://user-images.githubusercontent.com/9698542/159401862-ce98d2f4-8923-48d5-983e-3c545addc065.png)

Reply:
Copied from bot start and bot watch task:
<img width="615" alt="image" src="https://user-images.githubusercontent.com/9698542/159402212-c458e62a-9800-4b2f-be7b-de1b702b266b.png">